CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [6.0.0] - Unreleased
```

Comment:
You need to point this pr to prod if you want to immediately release a new version.

Reply:
I don't. See it's marked `Unreleased`. Because of breaking changes we already know the impact and could predefine the next version.

Reply:
I actually wanna add more rules that are web local now